src/billing/components/PayAsYouGo/CancellationOverlay.tsx

```diff
@@ -23,11 +30,42 @@ const CancellationOverlay: FC<Props> = ({onHideOverlay}) => {
   const {handleCancelAccount} = useContext(BillingContext)
   const [hasAgreedToTerms, setHasAgreedToTerms] = useState(false)
   const [hasClickedCancel, setHasClickedCancel] = useState(false)
+  const {
+    reason,
+    canContactForFeedback,
+    suggestions,
+    shortSuggestion,
+  } = useContext(CancelServiceContext)
+  const quartzMe = useSelector(getQuartzMe)
+  const org = useSelector(getOrg)
+
+  const sendDetailsToRudderstack = () => {
+    if (
+      !isFlagEnabled('rudderStackReporting') ||
+      !isFlagEnabled('trackCancellations')
+    ) {
+      return
+    }
+
+    const payload = {
+      org: org.id,
+      tier: quartzMe?.accountType,
+      email: quartzMe?.email,
+      alternativeProduct: shortSuggestion,
+      suggestions,
+      reason: VariableItems[reason],
+      canContactForFeedback: canContactForFeedback ? 'true' : 'false',
+    }
+
+    event('Cancel Org Executed', payload)
```

Comment:
we should DEFINITELY report the `reason` through event even if we're not sending this to rudderstack so that we can get access to this